doc/rules/class_notation/final_internal_class.rst

```diff
@@ -29,7 +29,7 @@ Default value: ``['@internal']``
 ~~~~~~~~~~~~~~~~~~~~~~
 
 Class level annotations tags that must be omitted to fix the class, even if all
-of the white list ones are used as well. (case insensitive)
+of the white list ones are used as well. (case insensitive).
```

Comment:
This looks odd.

Reply:
I hope you meant the dot before opening parenthesis.